samza-core/src/main/java/org/apache/samza/storage/StorageManagerUtil.java

```diff
@@ -323,7 +319,7 @@ public boolean storeExists(File storeDir) {
    * If the file does not exist, returns null.
    * // TODO HIGH dchen add tests at all call sites for handling null value.
    *
-   * @param storagePartitionDir base directory for the store
+   * @param storagePartitionDir base directory for the store checkpoint file
```

Comment:
"store directory to read the checkpoint file from"

"base directory" is overloaded, it's used to refer to the "logged-store" dir as well as the task store dir, but not the store checkpoint dir.